cli/src/diff_util.rs

```diff
@@ -1366,9 +1366,14 @@ pub fn show_file_by_file_diff(
                 ui,
                 writer.as_mut(),
                 tool,
+                temp_dir.path(),
                 &maplit::hashmap! {
-                    "left" => left_path.to_str().expect("temp_dir should be valid utf-8"),
-                    "right" => right_path.to_str().expect("temp_dir should be valid utf-8"),
+                    "left" => left_path
+                        .strip_prefix(temp_dir.path()).expect("path should be relative to temp_dir")
```

Comment:
nit: I would put this right after lines 1361-1362, or maybe even in the same statement. I think being closer to where the path is produced makes it clearer why it's a safe assumption. But I'm also fine with leaving it here if you prefer

Reply:
Thanks! That makes sense, and made the function look a little nicer, too. I'll squash (and rebase on main, oops).